src/screenbot.js

```diff
@@ -1,3 +1,9 @@
+var ERROR_CODES = {
```

Comment:
Actually, it may be easier to have these constants individually. There's not really any reason why they would need to be looped over.

```
var CLIENT_UNAVAILABLE = { code: 1, message: 'Screenbot isn\'t running' };
...
```
The names are probably unique enough as symbols. There's no one exact best way to define constants. It may be in an object if you need to access them dynamically by the string key. It may be in an array if you need to access them by index. In this case, it looks like just having the symbols is all we need so we might as well because it makes the code shorter to write just `CLIENT_UNAVAILABLE` over `ERROR_CODES.CLIENT_UNAVAILABLE`.

Also, it's probably fine to have these inside the `Screenbot` closure.